boudicca.events/eventcollectors/src/main/kotlin/events/boudicca/eventcollector/LocalCollectorDebug.kt

```diff
@@ -1,6 +1,6 @@
 package events.boudicca.eventcollector
 
-import base.boudicca.api.eventcollector.EventCollectorDebugger
+import base.boudicca.api.eventcollector.debugger.EventCollectorDebugger
```

Comment:
moving this class is a breaking change. since we are still in our 0.* versions i will allow it, but beginning at 1.* we cannot do this anymore.